src/pareto.js

```diff
@@ -0,0 +1,11 @@
+export default function(alpha) {
+    if (arguments.length != 1) {
+        throw new SyntaxError("pareto(alpha) must be called with only the alpha parameter.");
+    }
+    if (alpha < 0) {
+        throw new SyntaxError("pareto(alpha) must be called with alpha >= 0.");
+    }
+    return function() {
+        return 1.0 / Math.pow((1 - Math.random()), 1.0 / alpha);
```

Comment:
Replace `1.0` with `1` (since they are equivalent as JavaScript has only one number type).

Remove extra parens around `(1 - Math.random())`. I was tempted to replace that with just `Math.random()` but I see it does address the case where Math.random returns exactly zero.